building.md

```diff
@@ -17,13 +17,18 @@ Building above binaries require [golang](https://golang.org/dl/) with version `>
   ```
 - Verify binaries are installed
   ```sh
-  docker --help
-  # look for 
-  Management Commands:
-    generate*   Generate artifacts (CNCF Notary Project, 0.1.0)
-    notation*   Manage signatures on Docker images (CNCF Notary Project, 0.5.3-alpha)
-  
   which notation
-  # output
+  # expected output
   /home/<user>/bin/notation
   ```
+
+  If you confront `notation not found`, please add `~/bin/` to your $PATH:
+  ```sh
+  export PATH="$HOME/bin:$PATH"
+  ```
+  If you would like to add the path permanently, add the command to your `~/.profile` file:
+  ```sh
+  echo 'export PATH="$HOME/bin:$PATH"' >> ~/.profile
+  source ~/.profile
```

Comment:
Depending upon user's shell the location of profile will change. For Eg: 
* bash: `~/.bash_profile` or `~/..profile`
* zsh: `~/.zprofile` 
* ksh: `~/..profile`

Reply:
Updated.